build-system/pr-check/visual-diff-tests.js

```diff
@@ -30,21 +30,25 @@ const {runCiJob} = require('./ci-job');
 
 const jobName = 'visual-diff-tests.js';
 
+const baseCommand = 'gulp visual-diff';
+
 function pushBuildWorkflow() {
   downloadNomoduleOutput();
   timedExecOrDie('gulp update-packages');
   process.env['PERCY_TOKEN'] = atob(process.env.PERCY_TOKEN_ENCODED);
-  timedExecOrDie('gulp visual-diff --nobuild --master');
+  timedExecOrDie(`${baseCommand} --nobuild --master`);
 }
 
 function prBuildWorkflow() {
   process.env['PERCY_TOKEN'] = atob(process.env.PERCY_TOKEN_ENCODED);
   if (buildTargetsInclude(Targets.RUNTIME, Targets.VISUAL_DIFF)) {
     downloadNomoduleOutput();
     timedExecOrDie('gulp update-packages');
-    timedExecOrDie('gulp visual-diff --nobuild');
+    timedExecOrDie(`${baseCommand} --nobuild --esm`);
```

Comment:
Missing a call to `downloadModuleOutput()` after line 45, correct? (Same applies to line 36.)